libs/util/shared/src/lib/utils/strip-html.ts

```diff
@@ -0,0 +1,6 @@
+export const stripHtml = function (html: string): string {
+  if (!html) {
+    return ''
+  }
+  return html.replace(/<[^>]*>/g, '')
```

Comment:
I think we should rely on the `DOMParser` API instead, see: https://stackoverflow.com/questions/822452/strip-html-from-text-javascript/47140708#47140708
This should be more robust than a regex